data/process/archive/getAllCharactersStats.ipynb

```diff
@@ -0,0 +1,176 @@
+{
```

Comment:
Did you clear all processing before committing?  Doing so makes this file easier to manage over time.

Reply:
I created a folder called "archive" and moved the script that we no longer need there... but just to keep them for now...

Reply:
How big are these files and do they need to be a part of our repository?

Reply:
They are only 43k... I was thinking living them there until Monday once we are ready for cleanup, but I can remove them now if needed.